gandalf.js

```diff
@@ -1,29 +1,30 @@
-/**
```

Comment:
SCRIPT-This hardcoded value might be better suited as a constant at the top of the file.